apps/dsp-app/src/app/project/data-models/data-models.component.html

```diff
@@ -10,50 +10,37 @@
     </mat-icon>
   </div>
   <div *ngIf="isAdmin$ | async" class="action-buttons">
-    <button color="primary" mat-raised-button class="create" (click)="open('add-ontology')">
+    <a color="primary" mat-raised-button class="create" [routerLink]="['..', 'add-ontology']">
```

Comment:
use route constants, in this way we can track where this route is used in whole app